### ISSUE TITLE
Lot of notes are deleted after I reinstall chrome

### ISSUE THREAD

**Issue by movibeast** [phaistonian/Chrome-Notepad#35]
Lot of notes are deleted after I reinstall chrome, why doe it happens? or if it is not deleted, it don't show the about ~few weeks old saved

**Comment by barmanrajdeep** [comment]
are you logged into chrome browser? Being logged into your account in chrome browser keeps them synced

**Comment by movibeast** [comment]
Yes I am logged into the chrome and it sync everything, everytime but notepad notes are removed.
